module1/homework/removeVowels/vowel.cpp

```diff
@@ -0,0 +1,16 @@
+#include "vowel.hpp"
+
+#include <algorithm>
+#include <array>
+#include <iostream>
+
+void removeVowels (std::vector<std::string>& fullText) {
+
+const std::array<char,12> vowelArr {'a', 'A', 'e', 'E', 'i', 'I', 'o', 'O', 'u', 'U', 'y', 'Y'};
+
+    for (auto& element : fullText) {
+        for (const auto& vowel : vowelArr) {
```

Comment:
const auto, because vowel it's char :)
```suggestion
        for (const auto vowel : vowelArr) {
```

Reply:
+1

Reply:
It's our previous homework :)